### PR TITLE
fix download modal buttons

### PR DESCRIPTION
**Pull request by kubic71**

### CODE REVIEW

**Comment by kubic71** [comment]
On desktop:
image

Mobile:
image

**Review by KPrasch — APPROVED** [review]
This appears to fix the overflow/wrapping issue on my phone.  LGTM ⚡

**Comment by kubic71** [comment]
@KPrasch I also just fixed the color theme of the buttons in the modal.
Also added a link to each button (because you said "buttons all thud (lead to nowhere without indication)")

Maybe a re-review?

**Comment by kubic71** [comment]
image

**Comment by kubic71** [comment]
ezgif-2-90c2b77348

**Comment by kubic71** [comment]
Note: I also switched the order of css files included, and put our `style.css` after the Bootstrap.
  That makes more sense to me, as it allows us to customize the Boostrap styles more easily, without the need to resort to using `!important` for instance.

**Comment by KPrasch** [comment]
Okay, some time has passed here and this will need a rebase to be integrated.

File: src/index.html [hunk 1]
```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
                 <div class="modal-content">
                     <div class="modal-header">
                         <h5 class="modal-title" id="exampleModalLabel">Get Vowel Sounds, Encrypted</h5>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
```
Comment (kubic71) [inline src/index.html:85]: Removed this, beause we already have a dedicated "close" button, and this  extra button looked off-theme color-wise.

File: src/index.html [hunk 2]
```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
...
                     <div class="modal-body">
                         The ciphertext of the record is available for download.
                         Once the contract reaches 10ETH, the decryption key will become available.
                     </div>
                     <div class="modal-footer">
-                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                        <button type="button" class="btn btn-primary">IPFS</button>
-                        <button type="button" class="btn btn-primary">Arweave</button>
-                        <button type="button" class="btn btn-primary">Bittorrent</button>
+                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
```
Comment (kubic71) [inline src/index.html:91]: Turns out we need `data-bs-dismiss` instead of `data-dismiss` with the current bootstrap version

**Comment by kubic71** [comment]
Rebase onto the current main and should be working again :)

**Comment by jMyles** [comment]
Fixed #63

**Pull request merged**